powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthServiceImpl.java

```diff
@@ -261,13 +262,14 @@ private VerifySignatureResponse verifySignatureImplNonTransaction(VerifySignatur
         String applicationKey = request.getApplicationKey();
         String dataString = request.getData();
         String signature = request.getSignature();
+        String signatureVersion = request.getSignatureVersion();
```

Comment:
In the beginning of the `verifySignature` method in this class there is a request validation logic. Since the `signatureVersion` is a required parameter, let's add it into the validation logic.

See:
https://github.com/wultra/powerauth-server/blob/5b7914229ce22990abebc4a7cb008d1a3bb346c0/powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthServiceImpl.java#L278